epub33/rs/index.html

```diff
@@ -384,14 +384,18 @@ <h3>Network access</h3>
 			<section id="sec-epub-rs-external-links">
 				<h3>External links</h3>
 
-				<p data-tests="#pub-external-links">Reading systems SHOULD open links that resolve outside the <a>EPUB
-						publication</a> in a new browser instance to ensure that the browser's security and privacy
-					controls are available to users.</p>
+				<p><span>Reading systems SHOULD alert users before opening a link that declares a <a
+							data-cite="url#concept-url-scheme">scheme</a> [[url]] (i.e., that resolves to web content
+						outside the [=EPUB publication=] or requires a system application to open).</span>
+					<span data-tests="#pub-external-links">If the user consents to open the link, reading systems SHOULD
+						open it using the default application registered for the scheme (i.e., to help ensure
```

Comment:
I am not sure this makes sense. For instance, Chrome may open PDFs in Chrome, even if there is another default viewer for PDF. In fact, it would be very odd to recommend we open system default apps since browsers typical do *not* do that. And not all operating systems even have the concept of a default. For instance, Android will let you pick an app from list. The user can decide one is app is the one they always want to use, but that isn't required.